### DISCUSSION TITLE
Welcome to oc-blogtaxonomy-plugin Discussions!

### DISCUSSION THREAD

**Discussion by GinoPane** [GinoPane/oc-blogtaxonomy-plugin#82] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HitArrowLegend** [comment]
Hey Gino, glad I found about this Github's feature :)
About me, I'm mainly focusing on developping blogs about tons of things, and my two last projects and two next ones are relying heavily on your plugin to work 🥇 Loving your work so far 👯

**Reply by GinoPane** [reply]
Thank you! Much appreciated 😃
